src/ubuntu/18.04/debpkg/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/dotnet-buildtools-prereqs:ubuntu-18.04
+
+# Install the deb packaging toolchain we need to build debs
+RUN apt-get update \
+    && apt-get -y install \
+        debhelper \
+        build-essential \
+        devscripts \
+    && rm -rf /var/lib/apt/lists/*
+
+# liblldb is needed so deb package build does not throw missing library info errors
+# Package resource list is already set in the base Image /etc/apt/sources.list.d/llvm.list
+RUN apt-get update \
```

Comment:
I realize this is the previous pattern used - but is there a specific reason to install liblldb in it's own layer?

Reply:
I am not aware of any specific reason. Noticed that base layer installs liblldb-3.9-dev.

Reply:
My preference would be to combine the two.  It will simplify everything - build, pull, and startup time.

Reply:
Yes, Makes sense.

Reply:
I wasn't proposing the removal of the debpkg image.  I was proposing combining the two layers within the debpkg image.  I think keeping the debian packaging dependencies separated is valuable.